colossalai/communication/utils.py

```diff
@@ -107,3 +107,34 @@ def gather_split_1d_tensor(tensor):
     chunks = [gathered[i*numel:(i+1)*numel] for i in range(world_size)]
     dist.all_gather(chunks, tensor, group=gpc.get_group(ParallelMode.PARALLEL_1D))
     return gathered
+
+import os
+import torch
+import warnings
+def check_single_machine_multi_gpu_p2p_available(gpu_list):
+    unavailble = []
+    check_p2p_result = os.popen("./simpleP2P")
```

Comment:
how dose this file generated?

Reply:
The file located in /usr/local/cuda-10.1/samples/0_Simple/simpleP2P.

Reply:
We can use simpleP2P to test peer to peer after makefile.